### PR TITLE
#78 Re-enable `jupyterlab.browser_check`

### LINKED ISSUE #78: `jupyterlab.browser_check` is disabled
`browser_check` is broken with the changes in jupyter_server 2.0.2 (https://github.com/jupyter-server/jupyter_server/issues/1143). It is disabled in our CI and should be re-enabled once it's fixed upstream.

Related PR https://github.com/QuantStack/jupytercad/pull/77

### PR DESCRIPTION
**Pull request by arjxn-py**
Should potentially fix #78

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ jobs:
           jupyter cad --version
           # jupyterlab.browser_check will fail due to the core extension jupyterlab_pygments.
           # It does not support lab v4.0.0
```
Comment (arjxn-py) [inline .github/workflows/build.yml:119]:
```suggestion
```
Should I remove these comments?

**Review by arjxn-py — COMMENTED** [review]
Looks like build is passing nicely

File: .github/workflows/build.yml (hunk 1 above)
Comment (martinRenou) [reply .github/workflows/build.yml:119]: Sounds good!

Comment (arjxn-py) [reply .github/workflows/build.yml]: Done in [`b639bf7`](https://github.com/jupytercad/JupyterCAD/pull/386/commits/b639bf751c1ca0ca050d22c8ca35fe8956412f61)

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**